### PR TITLE
feat: Faster skeleton loading on drops

### LINKED ISSUE #10701: Skeleton loading on drops
image

## Question
- does all drops load simultaneously ? Because the skeleton is there till all the drops load
	- if so, would not it be better if they would load one by one, so the first one (newest), will load fastest

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #10701

## Needs Design check

- @exezbcz please review

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-08-05 at 13 35 46

### CODE REVIEW

File: components/drops/Drops.vue [hunk 1]
```diff
@@ -80,11 +82,11 @@ const CURRENT_DROP_STATUS = Object.values(DropStatus).filter(
 
 const { $i18n } = useNuxtApp()
 const { urlPrefix } = usePrefix()
-const { drops, loaded } = useDrops({
+const { drops, loaded, count } = useDrops({
   active: [true],
   chain: !isProduction ? [urlPrefix.value] : [],
-  limit: 100, // set limit to enable sort from backend
-})
+  limit: 100
```
Comment (preschian) [inline components/drops/Drops.vue:88]: seems like git hooks didnt work in some machine. I will revert back to husky

**Comment by exezbcz** [comment]
image

is it really faster?

**Comment by hassnian** [comment]
@exezbcz adding new drops broke the grid, fixed it , pls try again

**Comment by exezbcz** [comment]
yea that loads quick, thanks!

**Review by vikiival — APPROVED** [review]

**Pull request merged**